pkg/disk/nodeserver.go

```diff
@@ -510,6 +510,11 @@ func (ns *nodeServer) NodeStageVolume(ctx context.Context, req *csi.NodeStageVol
 	targetPath := req.StagingTargetPath
 	// targetPath format: /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pv-disk-1e7001e0-c54a-11e9-8f89-00163e0e78a0/globalmount
 
+	if err := os.MkdirAll(targetPath, 0755); err != nil {
```

Comment:
I remember we can't put ensureDir before the runD check, is there any particular reason why we have to put it there?